Cargo.toml

```diff
@@ -42,7 +42,7 @@ chrono = { version = "0.4", features = ["serde"] }
 failure = "0.1"
 x509-parser = "0.6.0"
 sspi = "0.3.0"
-ironrdp = "0.4"
+ironrdp = { git = "https://github.com/Devolutions/IronRDP", branch = "preconnection-pdu-improvements" }
```

Comment:
Temporary change before preconnection-pdu-improvements IronRDP branch is berged to the master and pushed to the crates.io